### PR TITLE
Fix F score formula and etc

### PR DESCRIPTION
**Pull request by maoxc**

### CODE REVIEW

**Comment by maoxc** [comment]
F-score formula didn't make sense when I let B=1 :)

File: slides/05/05.md [hunk 1]
```diff
@@ -285,7 +285,7 @@ The $F_1$ score can be generalized to $F_β$ score, where recall is $β$ times
 more important than precision; $F_2$ favoring recall and $F_{0.5}$ favoring
 precision are commonly used.
 
-$$ F_β = \frac{(1 + β)^2 ⋅ \mathrm{precision} ⋅ \mathrm{recall}}{β^2 ⋅ \mathrm{precision} + \mathrm{recall}}
+$$ F_β = \frac{(1 + β^2) ⋅ \mathrm{precision} ⋅ \mathrm{recall}}{β^2 ⋅ \mathrm{precision} + \mathrm{recall}}
        = \frac{\hphantom{()}β^2\mathit{TP~~~~~}+\mathit{~~~~~TP}}{β^2(\mathit{TP}+\mathit{FP})+\mathit{TP}+\mathit{FN}}.$$
```
Comment (maoxc) [inline slides/05/05.md:289]: I think on wikipedia the denominator is different.. maybe switch FP and FN?

Comment (foxik) [reply slides/05/05.md:289]: Oh, true. I even have the correct form on the paper, where I counted it, but then I put β^2 to an incorrect place. Will fix.

**Comment by foxik** [comment]
Thanks, merging, and will fix the position of β^2 after.

**Pull request merged**